nncf/common/pruning/statistics.py

```diff
@@ -161,26 +161,26 @@ class PrunedModelTheoreticalBorderline(Statistics):
     def __init__(self,
                  num_pruned_layers: int,
                  num_prunable_layers: int,
-                 max_prunable_flops: float,
-                 max_prunable_params: float,
+                 min_prunable_flops: float,
+                 min_prunable_params: float,
```

Comment:
Seems like `max_prunable_flops` makes more sense. 

But you can rename them as following, for example:
```suggestion
                 min_flops: float,
                 min_params: float,
```

Reply:
`max_prunable_flops` is not quite correct, since this value represents the minimal FLOPS we are able to reach with the maximum compression rate. I don't have problems with renaming it to `min_flops` though.

Reply:
Possible, it depends on an interpretation. My opinion is the max prunable flops is number of flops when prunable flops were pruned to their maximum.

Reply:
Renamed to `min_possible_*`. It is a good candidate and the variables are already used with that name inside `PrunedModelTheoreticalBorderline`.